boards/common/nrf52/Makefile.include

```diff
@@ -22,4 +22,6 @@ else ifeq (openocd,$(PROGRAMMER))
   # build from source (master > 2018, August the 13rd) is required.
   DEBUG_ADAPTER ?= jlink
   OPENOCD_CONFIG = $(RIOTBOARD)/common/nrf52/dist/openocd.cfg
+else ifeq (dfu-util,$(PROGRAMMER))
+  DFU_USB_ID ?= ${USB_VID}:${USB_PID}
```

Comment:
can we move this to a common place?

Reply:
Sure !

Reply:
Move it to `makefiles/tools/dfu-util.inc.mk`